test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,9 @@
 const { humanCatDogYears } = require("../src");
 
-// Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
+	test("returns array of human, cat, dog years", () => {
+		expect(humanCatDogYears(10)).toEqual([10, 56, 64]);
+		expect(humanCatDogYears(3)).toEqual([3, 28, 29]);
+		expect(humanCatDogYears(1)).toEqual([1, 15, 15]);
```

Comment:
It would be good to add a test case for an input number of 2 since your code covers that logic, how do you know if its working otherwise?